packages/core/src/components/MenuButton/MenuButton.tsx

```diff
@@ -373,7 +374,7 @@ const MenuButton: VibeComponent<MenuButtonProps> & {
     const tooltipNode = (tooltipChildren: React.ReactElement) => (
       <Tooltip
         content={overrideTooltipContent}
-        position={tooltipPosition}
+        position={tooltipPosition as TooltipPositionsType}
```

Comment:
same as before, not sure why the ts cast is needed here

Reply:
Mistake